com.unity.render-pipelines.high-definition/Runtime/Material/Lit/LitPathTracing.hlsl

```diff
@@ -73,12 +75,10 @@ bool CreateMaterialData(PathIntersection pathIntersection, BuiltinData builtinDa
     else // Below
     {
         float NdotV = -dot(mtlData.bsdfData.normalWS, mtlData.V);
-        float F = F_FresnelDielectric(1.0 / mtlData.bsdfData.ior, NdotV);
+        float F = F_FresnelDielectric(1.0 / mtlData.ior, NdotV);
 
         // If N.V < 0 (can happen with normal mapping) we want to avoid spec sampling
         bool consistentNormal = (NdotV > 0.001);
-        mtlData.bsdfWeight[0] = 0.0;
```

Comment:
is it expected to not fill mtlData.bsdfWeight[0] anymore? (I do'nt have the full picture of the code, so maybe it is init elsewhere)

Reply:
Yes it it, right above.